proxy/src/main/java/com/velocitypowered/proxy/protocol/packet/chat/keyed/KeyedChatHandler.java

```diff
@@ -105,24 +88,14 @@ private Function<PlayerChatEvent, MinecraftPacket> handleOldSignedChat(KeyedPlay
     return pme -> {
       PlayerChatEvent.ChatResult chatResult = pme.getResult();
       if (!chatResult.isAllowed()) {
-        if (playerKey.getKeyRevision().noLessThan(IdentifiedKey.Revision.LINKED_V2)) {
-          // Bad, very bad.
-          invalidCancel(logger, player);
-        }
-        return null;
+          return null;
```

Comment:
I know it was a pull request made by mistake, but you still shouldn't do this, it causes more problems than it "solves", there are plugins like SignedVelocity that partially solve the problem until proper support is given in Velocity soon

Reply:
Thank you for commenting, nonetheless. The removal of cancellation is more or less a biased figure of a client of mine that I work with who refuses to install this resource. I did notice that you were working on a good solution for this, and I greatly appreciate your fantastic contributions. I'll certainly use this as a POR (point of reference) to the owner of the network that uses this fork.

Reply:
Ah, lastly, do you mind if I personally contact you about a relatively problematic issue that my fork manages to resolve unideally? I feel it's better to discuss this elsewhere rather than in an accidental/closed PR. It's in relation to these changes I've made here: 
![image](https://github.com/PaperMC/Velocity/assets/36140389/658f030c-0be1-4e51-b9f5-0afd606d4e4e)